drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -908,6 +908,11 @@ zigbeeManufacturer:
     manufacturer: Samsung Electronics
     model: SAMSUNG-ITM-Z-002
     deviceProfileName: rgbw-bulb-2700K-5000K
+  - id: Samsung/SAMSUNG-ITM-Z-005
+    deviceLabel: Samsung Switch
+    manufacturer: Samsung Electronics
+    model: SAMSUNG-ITM-Z-005
+    deviceProfileName: switch-color-temp
```

Comment:
Make sure you change the `deviceProfileName` to match your new `switch-color-temp-2700K-5000K` profile :)

Reply:
Thank you for your reply I missed it :) But I faced an error that I can't chage it directly. Could you guid me how I can make it, please? I wonder if I make a new branch and make a new pull request for merge. Thank you in advance. :)

Reply:
That is strange, I am not sure why you cannot make the changes. You can try using `git pull` to sync your local branch with the remote branch on GitHub, but if that doesn't work, you can close this PR and make a new branch and a new PR.

I am wondering if part of the issue is that your branch is called `RaihaPark:main`, so maybe it got out of sync with the remote `SmartThingsCommunity:main` branch. If you end up creating a new PR with a new branch, try naming your branch something like `samsung-zigbee-switch` or something like that so that is clear what is being added in the new branch. I hope that helps :)

Reply:
Thank you for your reply and kindness guidance. I changed it today. It is really strange that why I can do it today but coudn't did that 2 days ago :)